library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -452,7 +447,7 @@ private void scan(Path filePath, Path rootPath, Map<String, Long> lastModifiedTi
           }
         }
       } catch (FileNotFoundException fnf) {
-        LOG.warn("Failed to list directory {}", filePath, fnf);
+        LOG.error("Failed to list directory {}", filePath, fnf);
```

Comment:
If it is an error, shouldn't the exception be propagated? Otherwise, if it is an acceptable condition, why not log at INFO?